### ISSUE TITLE
Work on directories option

### ISSUE THREAD

**Issue by Auscobe** [GPUOpen-Effects/FidelityFX-CLI#12]
It's possible to use a loop to upscale a folder full of images, but this is pretty slow for what it could be. Could you make it able to work on folders, probably by not restarting itself between frames?

Helpful for:
Upscaling a video's worth of frames
Upscaling a lot of images that don't relate to each other.
This would be drastically sped up in either case.

Thanks in advance regardless of decision.

**Comment by adam-sawicki-a** [comment]
The tool can process multiple files in one launch, without restarting the entire framework with DirectX etc. All you need to do is to pass parameters like:

```
FidelityFX_CLI.exe [Options] <SrcFile1> <DstFile1> <SrcFile2> <DstFile2> ...
```

See README.md for details of the command-line syntax.

**Comment by Auscobe** [comment]
Yes, but that can result in a massive command. Wouldn't it be easier to be able to do it without doing that?
